boost.sh

```diff
@@ -119,6 +120,9 @@ OPTIONS:
     -linux
         Build for the Linux platform.
 
+    -linux-cxx-11-abi
+        Build for the Linux platform with gcc-4.8 ABI compatibility.
```

Comment:
Maybe `-linux-gcc-4.8-abi` or `-linux-gcc-4-abi`?

Reply:
These gcc4 variants could also be added as Linux architectures. I opted to keep them separate.